extensions/ql-vscode/src/remote-queries/remote-queries-manager.ts

```diff
@@ -52,14 +54,19 @@ export class RemoteQueriesManager {
     const executionEndTime = new Date();
 
     if (queryResult.status === 'CompletedSuccessfully') {
-      const resultIndexItems = await this.downloadResultIndex(credentials, query);
+      const resultIndex = await getRemoteQueryIndex(credentials, query);
+      if (!resultIndex) {
+        await showAndLogErrorMessage(`There was an issue retrieving the result for the query ${query.queryName}`);
```

Comment:
Is there a reason to wait for the popup to close?
```suggestion
        void showAndLogErrorMessage(`There was an issue retrieving the result for the query ${query.queryName}`);
```